salt/cloud/clouds/azurearm.py

```diff
@@ -221,7 +221,10 @@ def _cache(bank, key, fun, **kwargs):
     items = cache.fetch(bank, key)
     if items is None:
         items = {}
-        item_list = fun(**kwargs)
+        try:
+            item_list = fun(**kwargs)
+        except CloudError as exc:
+            log.debug('There was a cloud error calling {0} with kwargs {1}: {2}'.format(fun, kwargs, exc))
```

Comment:
Are you sure you want an error like this down in the debug log and not more prominent?